src/datasets/builder.py

```diff
@@ -679,6 +685,12 @@ def download_and_prepare(
         >>> ds = builder.download_and_prepare("s3://my-bucket/my_rotten_tomatoes", storage_options=storage_options, file_format="parquet")
         ```
         """
+        if use_auth_token != "deprecated":
+            warnings.warn(
+                "'use_auth_token' was deprecated in version 2.7.1 and will be removed in 3.0.0. Pass `use_auth_token` to the initializer/`load_dataset_builder` instead.",
+                FutureWarning,
+            )
```

Comment:
We can deprecate it, but it should still work if some users were passing it here:

```suggestion
            )
        else:
            use_auth_token = self.use_auth_token
```